thoth-api/src/graphql/model.rs

```diff
@@ -1461,6 +1466,48 @@ impl QueryRoot {
     fn reference_count(context: &Context) -> FieldResult<i32> {
         Reference::count(&context.db, None, vec![], vec![], vec![], None).map_err(|e| e.into())
     }
+
+    #[graphql(description = "Query a title by its ID")]
+    fn title(context: &Context, title_id: Uuid) -> FieldResult<Title> {
+        Title::from_id(&context.db, &title_id).map_err(|e| e.into())
+    }
+
+    #[graphql(description = "Query titles by work ID")]
+    fn titles(
+        context: &Context,
+        #[graphql(default = 100, description = "The number of items to return")] limit: Option<i32>,
+        #[graphql(default = 0, description = "The number of items to skip")] offset: Option<i32>,
+        #[graphql(
+            default = "".to_string(),
+            description = "A query string to search. This argument is a test, do not rely on it. At present it simply searches for case insensitive literals on institution_name, ror and institution_doi"
```

Comment:
The description should match the fields that are queried (title, subtitle, full_title)